app/src/main/java/com/pajato/android/gamechat/common/PlayModeManager.java

```diff
@@ -88,15 +131,60 @@ public void dismissPlayModeMenu() {
      * Create and show the play mode popup menu. The popup menu must be created with the anchor in
      * the current fragment layout, so it cannot be shared across fragments.
      */
-    public void showPlayModeMenu(View anchorView) {
-        mPlayModePopup = new PopupMenu(anchorView.getContext(), anchorView);
-        mPlayModePopup.getMenuInflater().inflate(R.menu.player2_menu, mPlayModePopup.getMenu());
-        mPlayModePopup.setOnMenuItemClickListener(mListener);
-        mPlayModePopup.show();
+    public void showPlayModeMenu(Activity activity, View anchorView) {
```

Comment:
I think this method might be trimmed down by applying similar logic to my previous comments.